src/translations/sqlb_pt_BR.ts

```diff
@@ -2860,7 +2860,7 @@ x~y	Intervalo: valores entre x e y
     <message>
         <location filename="../MainWindow.ui" line="1869"/>
         <source>E&amp;xit</source>
-        <translation>&amp;Sair</translation>
+        <translation>Sair</translation>
```

Comment:
Why did you disable the shortcut?

Reply:
It seems sensible, specially since there are already OS and our Control-Q to exit the program. In any case, just for studying other solutions, what are other programs in Portuguese doing? Except Import, the other File menu entries seem pretty usual. Are they using I? Are there cases where S is reused for Save and Exit? Note that when the letter is repeated, you can still cycle between the instances and then press Enter. Another option is the one used in Asiatic languages: using a parenthesized letter as accelerator, as in "Sair (X)", but I haven't seen it used in Occidental languages.